release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -2936,6 +2936,10 @@ duplicates and some are misleading in that they match wrong keys in the
 linux-audit dictionary.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1496176[*BZ#1496176*])
 
+*  NFS provisioning of the Quick Installer should be limited to hosts that are
```

Comment:
I'd replace the first sentence with "Limit NFS provisioning of the Quick Installer to hosts that are part of the cluster."